styleguide/derek/_partials/solutions/testimonials.ejs

```diff
@@ -1,26 +1,47 @@
+<%
+// While seemingly redundant, this is actually necessary for EJS to evaluate variable scope.
+var isSubpanel = !!isSubpanel;
```

Comment:
Then the reason this is here is to make up for the fact that `isSubpanel` is undefined in most cases.

If you are so inclined you can set it globally to `false` in the `_harp.json` globals:

https://github.com/rackerlabs/zoolander/blob/c04259429f2fdb5e18b4055626d0f4371c90d7af/styleguide/_harp.json#L2